homeassistant/components/netatmo/climate.py

```diff
@@ -161,12 +163,7 @@ def target_temperature(self):
     @property
     def current_operation(self):
         """Return the current state of the thermostat."""
-        state = self._data.room_status[self._room_id]['heating_status']
-        if state is False:
-            return STATE_IDLE
-        if state is True:
-            return STATE_HEAT
-        return STATE_UNKNOWN
+        return self.operation_mode
```

Comment:
```suggestion
        return self._operation_mode
```

Then remove `operation_mode` property.

Reply:
Resolved in 287de20.